sklearn/neighbors/binary_tree.pxi

```diff
@@ -1130,15 +1130,15 @@ cdef class BinaryTree:
                 self.idx_array_arr,
                 self.node_data_arr,
                 self.node_bounds_arr,
+                self.sample_weight_arr,
```

Comment:
Is this reordering for any functional reason

Reply:
@jnothman No not any functional reason. Just thought of writing that variable with other _arr variables